### PR TITLE
Add JS help page, set Consolas as explicit font

### PR DESCRIPTION
**Pull request by pakrym**
Move the help under so people don't think it's a language selector.

image

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Client/css/site.scss [hunk 1]
```diff
@@ -213,7 +213,7 @@ form.comment {
 
 .code-line {
     white-space: pre-wrap;
-    font-family: monospace;
+    font-family: Consolas, monospace;
```
Comment (JoshLove-msft) [inline src/dotnet/APIView/APIViewWeb/Client/css/site.scss:216]: what is this change for?

Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:216]: So firefox uses Consolas by default. Chrome was using it already.

Comment (bterlson) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:216]: The default monospace font on some browser+OS combinations isn't particularly pleasant (e.g. FF+Windows is Courier New 😥). This helps for Windows users. (Adding Menlo would help Mac users).

Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:216]: Wikipedia says:
>It is not installed as a user-accessible typeface by default, although it is included with and used by Xcode.

**Pull request merged**